### ISSUE TITLE
Null belegs (monats/sammel/jahres)

### ISSUE THREAD

**Issue by petrovcicp** [a-sit-plus/at-registrierkassen-mustercode#378]
Hi all,
I don't speak german so I will ask this in english ... I hope you all don't mind this :-).

I have a question regarding null belegs. I understand well the technical nature of these belegs.

- they are belegs with zero amounts for all taxations 
- they logically don't update turnover counter
- they contain a normal chain value to last issued receipt
- they are signed like every normal receipt

But I simply don't get a point of these belegs. What are actually their value to financial office? What do we prove (or not prove) if we issue these belegs?
Usually when I don't get the point of something there is pretty much chance that I missed some parts of informations ... so, that's why I am asking this :)

Other main questions:
- Is it mandatory or is it user's choice to issue monatsbeleg every month?
- Same question for yahresbeleg
- Do we need to delivery these null belegs somewhere (financial office)?

I believe that for most of you who speak german as native language these questions will be trivial so thank you in advance for answers.

Peter

**Comment by norbert2** [comment]
Monatsbeleg HAS to be printed and kept in evidence.
At the end of the year the Monatsbeleg is automatically the Jahresbeleg.
This Jahresbeleg has to be checked as well as the Startbeleg with  an App. 
All the other Null-Belege don't have to be delivered somewhere. But also kept in evidence.
A Standard-Null-Beleg can be scanned if an inspechtion is taking place. After the scan of the QR-Code the inspector can see, if the chash-register  and the signature-unit is registered properly at FON.

**Comment by ErichFreitag** [comment]
You are right with your view to these receipts. Small correction: null-receipts also update the turnover counter but as x + 0 = 0 it does not change it really.

Their sense and meaning:
- start receipt - puts cash-box into operation, sets turnover counter to 0, inits DEP
- month receipt - to formally close a month, also used for formally closing the DEP backup
- year receipt - same as month plus check at finance-online or belegcheck-app
- check receipt - for financial authorities when they request one
- collection receipt - to indicate start and end of signature device failures
- close receipt - to formally close a cash-box's life

So the null receipts have more formal then financial reasons.

Finance-Online (or via Belegcheck app) receives start receipt and each year receipt. Year receipts have to be generated each start of year, month receipts have to be generated each month doing business.

**Comment by petrovcicp** [comment]
Norbert and Erich,
thank you for your simple explanation. 

Peter
